charts/manticoresearch/templates/manticore-worker.yaml

```diff
@@ -75,6 +75,8 @@ spec:
             value: {{ .Values.worker.logLevel }}
           - name: INSTANCE_LABEL
             value: {{ .Release.Name }}
+          - name: TZ
+            value: {{ .Values.worker.clusterName }}
```

Comment:
* The timezone should be exposed in the cluster configuration as a separate variable (not tied to `clusterName`).
* Before passing the variable to the container, check that it `is not empty`. Only then should it be applied.